### PR TITLE
Add keep alive getters and setters

### PR DESCRIPTION
**Pull request by dentarg**
Added tests for other get and setters for completeness.

### CODE REVIEW

File: spec/lib/ecco/client_spec.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    before  { client.set_server_id(server_id) }
+    subject { client.get_server_id }
+
+    it { is_expected.to eq(server_id) }
+  end
+
+  describe "#is_keep_alive" do
+    subject { client.is_keep_alive }
+
+    context "when truthy" do
```
Comment (jage) [inline spec/lib/ecco/client_spec.rb]: Shouldn't this say "when true" since we're not setting nor checking for truthyness?

Comment (dentarg) [inline spec/lib/ecco/client_spec.rb]: I didn't think it was that strict! But sure.

Comment (jage) [inline spec/lib/ecco/client_spec.rb]: I was expecting an rspec truthy-matcher (`be_truthy`), that's why this caught my attention.

**Comment by dentarg** [comment]
I opted to not update the README example because we now have illustrative specs.

**Comment by walro** [comment]
Will there be an integration test for ensuring that our client dies if we don't enable keepalive? Or will it be done in a separate PR?

**Comment by dentarg** [comment]
> Will there be an integration test for ensuring that our client dies if we don't enable keepalive? 

It's the other way around, keep alive is on by default

> Or will it be done in a separate PR?

Not sure how to do it, so yes, another PR if we do it.

**Comment by walro** [comment]
> It's the other way around, keep alive is on by default

Ah yes, should have worded that better.

> Not sure how to do it, so yes, another PR if we do it.

Me neither, so LGTM!

**Pull request merged**